pkg/kubelet/allocation/state/state.go

```diff
@@ -51,11 +53,13 @@ type Reader interface {
 	GetContainerResources(podUID types.UID, containerName string) (v1.ResourceRequirements, bool)
 	GetPodResourceInfoMap() PodResourceInfoMap
 	GetPodResourceInfo(podUID types.UID) (PodResourceInfo, bool)
+	GetPodLevelResources(podUID types.UID) (*v1.ResourceRequirements, bool)
```

Comment:
(tiny nit) the analogous `GetContainerResources` returns `(v1.ResourceRequirements, bool)`, should we keep it consistent and have the same return values?